autopush_rs/src/client.rs

```diff
@@ -733,17 +733,15 @@ where
                 }
             }
             Either::A(ClientMessage::Register { channel_id, key }) => {
+                data.srv.metrics.incr("ua.command.register").ok();
```

Comment:
there was also an associated log.info here in python

Reply:
Yup, we never used it and I don't know why we bothered logging it.